serverManager/service/include/ConfigHelper.h

```diff
@@ -50,11 +50,14 @@ class ConfigHelper
 #ifdef RIALTO_ENABLE_CONFIG_FILE
 private:
     void readConfigFile(const std::string &filePath);
+    void mergeEnvVariables();
 #endif // RIALTO_ENABLE_CONFIG_FILE
 
 private:
     std::unique_ptr<IConfigReaderFactory> m_configReaderFactory;
     std::map<std::string, std::string> m_sessionServerEnvVars;
+    std::map<std::string, std::string> m_jsonEnvVars;
```

Comment:
Just a suggestion....
I'm wondering if there's a better name, because I presume that these settings are no longer in json format when stored in the map?
Perhaps envVarsFromConfigFile ? (but feel free to decide)

Reply:
If you know better name, suggest one here :)

Reply:
changed.